Libraries/test/TestServerlessApp/serverless.template

```diff
@@ -2,6 +2,25 @@
   "AWSTemplateFormatVersion": "2010-09-09",
   "Transform": "AWS::Serverless-2016-10-31",
   "Description": "An AWS Serverless Application.",
+  "Parameters": {
+    "ArchitectureTypeParameter": {
+      "Type": "String",
+      "Default": "x86_64",
+      "AllowedValues": [
+        "x86_64",
+        "arm64"
+      ]
+    }
+  },
+  "Globals": {
+    "Function": {
+      "Architectures": [
+        {
+          "Ref": "ArchitectureTypeParameter"
+        }
+      ]
+    }
+  },
```

Comment:
Should we add these JSON blobs by default in the [CloudFormationJsonWriter.cs](https://github.com/aws/aws-lambda-dotnet/blob/feature/annotations/Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/Writers/CloudFormationJsonWriter.cs)?

Reply:
this is feature we have to support at some point to later to able to configure the arch of the lambda functions. 

Current code change is just a workaround to be able to run the test app on both x86 and ARM.